openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -210,47 +208,157 @@ pipeline {
         }
       }
     }
-    stage("Run App Tests") {
-      agent any
+    stage("FRONTEND Tests") {
+      agent {
+        label 'jenkins-slave-npm'
```

Comment:
I'm curious how this works?

Reply:
Yeah, it took a while to learn. Quite the learn curve. I can go over details on our next Dev Training session